README.md

```diff
@@ -3,6 +3,17 @@
 `s3-batch-object-store` is a Go module that allows for batch uploading of objects to a single S3 file and retrieving 
 each object separately using the AWS S3 API, fetching only the bytes for that specific object.
 
+The method basically consists of appending multiple objects to a single file, keep the information of where each object
+is placed in the file, and then upload one single file to s3 with many objects in it, reducing drastically the number 
+of PUT operations needed to store a large number of objets.
+
+Then, when you need to retrieve an object, you can use the index information to fetch that object and the GET call to s3
```

Comment:
should we add something about storing on the index data being beyond the scope of what this repo does, e.g.

```
The best solution for how to store and query your index data will depend on your application data patterns and is beyond the scope of what this package aspires to do. 
```

Reply:
good point, will add!